psalm-baseline.xml

```diff
@@ -553,6 +554,7 @@
     </ArgumentTypeCoercion>
     <InvalidArgument>
       <code>$columnDef</code>
+      <code>$mapping</code>
```

Comment:
The new entries correspond to drivers (XMLDriver and AttributeDriver), when calling `mapEmbedded()`. An improvement could be to add some checks in the drivers regarding the types.